src/EnergyPlus/NodeInputManager.cc

```diff
@@ -1008,223 +974,209 @@ void CalcMoreNodeInfo(EnergyPlusData &state)
-            ReportDewPoint = true;
-        }
-        if (NodeSpecificHeatRepReq(iNode) && NodeSpecificHeatSchedPtr(iNode) > 0) {
-            ReportSpecificHeat = (GetCurrentScheduleValue(state, NodeSpecificHeatSchedPtr(iNode)) > 0.0);
-        } else if (NodeSpecificHeatRepReq(iNode) && NodeSpecificHeatSchedPtr(iNode) == 0) {
-            ReportSpecificHeat = true;
+    if (state.dataNodeInputMgr->anyMoreNodeCalcsReq) {
+        for (int iNode = 1; iNode <= state.dataLoopNodes->NumOfNodes; ++iNode) {
+            auto &thisNode = state.dataLoopNodes->Node(iNode);
+            auto &thisMoreNodeInfo = state.dataLoopNodes->MoreNodeInfo(iNode);
+            if (thisNode.FluidType == DataLoopNode::NodeFluidType::Air) {
+                bool ReportWetBulb = false;
+                bool ReportRelHumidity = false;
+                bool ReportDewPoint = false;
+                bool ReportSpecificHeat = false;
+                if (thisNode.SPMNodeWetBulbRepReq) {
+                    ReportWetBulb = true;
+                } else if (thisMoreNodeInfo.wetBulbRepReq) {
+                    if (thisMoreNodeInfo.wetBulbSchedPtr > 0) {
+                        ReportWetBulb = (ScheduleManager::GetCurrentScheduleValue(state, thisMoreNodeInfo.wetBulbSchedPtr) > 0.0);
+                    } else {
+                        ReportWetBulb = true;
+                    }
+                }
+                if (thisMoreNodeInfo.relHumidityRepReq) {
+                    if (thisMoreNodeInfo.relHumiditySchedPtr > 0) {
+                        ReportRelHumidity = (ScheduleManager::GetCurrentScheduleValue(state, thisMoreNodeInfo.relHumiditySchedPtr) > 0.0);
+                    } else {
+                        ReportRelHumidity = true;
+                    }
+                }
+                if (thisMoreNodeInfo.dewPointRepReq) {
+                    if (thisMoreNodeInfo.dewPointSchedPtr > 0) {
+                        ReportDewPoint = (ScheduleManager::GetCurrentScheduleValue(state, thisMoreNodeInfo.dewPointSchedPtr) > 0.0);
+                    } else {
+                        ReportDewPoint = true;
+                    }
+                }
+                if (thisMoreNodeInfo.specificHeatRepReq) {
+                    if (thisMoreNodeInfo.specificHeatSchedPtr > 0) {
+                        ReportSpecificHeat = (ScheduleManager::GetCurrentScheduleValue(state, thisMoreNodeInfo.specificHeatSchedPtr) > 0.0);
+                    } else {
+                        ReportSpecificHeat = true;
+                    }
+                }
+                if (ReportWetBulb) {
+                    // if Node%Press was reliable could be used here.
+                    thisMoreNodeInfo.WetBulbTemp = Psychrometrics::PsyTwbFnTdbWPb(
+                        state, thisNode.Temp, thisNode.HumRat, state.dataEnvrn->OutBaroPress, thisMoreNodeInfo.reportingString);
+                } else {
+                    thisMoreNodeInfo.WetBulbTemp = 0.0;
+                }
```

Comment:
Aren't these already initialized to 0? Why set this again?

    struct MoreNodeData
    {
        Real64 WetBulbTemp = 0.0;